README.md

```diff
@@ -74,15 +74,31 @@ AWS$ sam local start-api
 AWS$ curl http://localhost:3000/
 ```
 
+The application expects two environment variables:
+ * `bareHost` defines the source of the Authority data (utvikle-a.bibsys.no for development, authority.bibsys.no for production)
+ * `bareApiKey` should be defined in the AWS SecretsManager and is needed to for update/PUT functionality
+
+```yaml
+      Environment: # More info about Env Vars: https://github.com/awslabs/serverless-application-model/blob/master/versions/2016-10-31.md#environment-object
+        Variables:
+          bareHost: "utvikle-a.bibsys.no"
+          bareApiKey: '{{resolve:ssm:bareApiKey:1}}'
+```
+
 The SAM CLI reads the application template to determine the API's routes and the functions that they invoke. The `Events` property on each function's definition includes the route and method for each path.
 
 ```yaml
       Events:
-        NvaBareProxy:
-          Type: Api
+        PostEvent:
+          Type: Api # More info about API Event Source: https://github.com/awslabs/serverless-application-model/blob/master/versions/2016-10-31.md#api
           Properties:
             Path: /authority
             Method: post
+        PutEvent:
+          Type: Api # More info about API Event Source: https://github.com/awslabs/serverless-application-model/blob/master/versions/2016-10-31.md#api
+          Properties:
+            Path: /authority/{scn}
+            Method: put
```

Comment:
Dersom du definerer en RequestModel (se https://github.com/awslabs/serverless-application-model/blob/master/versions/2016-10-31.md#function-request-parameter-object) så vil swagger spec for frontend bli bedre.